src/OpenFeature/Providers/Memory/Flag.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using OpenFeature.Constant;
+using OpenFeature.Error;
+using OpenFeature.Model;
+
+namespace OpenFeature.Providers.Memory
+{
+    /// <summary>
+    /// Flag representation for the in-memory provider.
+    /// </summary>
+    public class Flag
+    {
+
+    }
```

Comment:
Why do we need this empty class? Might be an `interface` or `abstract class`?

Reply:
Good call. We just need a non-generic base but I can use an interface.

Reply:
Made interface.